drivers/SmartThings/zigbee-switch/profiles/switch-power-energy-consumption-report-aqara-t1.yml

```diff
@@ -0,0 +1,29 @@
+name: switch-power-energy-consumption-report-aqara-t1
+components:
+  - id: main
+    capabilities:
+      - id: switch
+        version: 1
+      - id: powerMeter
+        version: 1
+      - id: energyMeter
+        version: 1
+      - id: powerConsumptionReport
```

Comment:
My understanding is that this capability is needed for ST Energy, and this is not supported in CN. What is the user experience here then? or am I incorrect?

Reply:
the ST Energy tool simply doesn't get anything from this device

it's never been visible on individual device cards